gatsby-browser.js

```diff
@@ -20,6 +22,26 @@ const checkDomain = () => {
     )
 }
 
+const updateURLAsPerUserLanguage = () => {
```

Comment:
@fasihali-deriv Would you please extract this function into a utility file and write unit tests for it 🙏🏻

Reply:
Hi @fasihali-deriv @farzin-deriv  let's move the function to utility. As tests are not yet setup, we are good to go for QA after this.